### ISSUE TITLE
`prefer-prototype-methods` false positive

### ISSUE THREAD

**Issue by GuoSirius** [sindresorhus/eslint-plugin-unicorn#1328]
```js
export default {
  name: 'Demo',
  created() {
    this.refreshDatetime = this.startClock.bind(this)
  },
  methods: {
    startClock() {}
  }
}
```

![image](https://user-images.githubusercontent.com/48358523/120572534-b3ebcb80-c44e-11eb-8af0-ccceb34e95e1.png)

**Comment by GuoSirius** [comment]
![image](https://user-images.githubusercontent.com/48358523/120572744-12b14500-c44f-11eb-9142-4f1ca501262a.png)

![image](https://user-images.githubusercontent.com/48358523/120572769-1ba21680-c44f-11eb-952c-20a93a2d647d.png)

**Comment by fisker** [comment]
Yes, this is bug.

Another case I found myself,

```js
const join = path.join.bind(path, __dirname);
```

My suggestion is to use this as a workaround for now.

```js
this.refreshDatetime = (...args) => this.startClock(...args)
```

**Comment by fisker** [comment]
Stop pasting snapshots, just copy your code.

**Comment by GuoSirius** [comment]
This rule is temporarily turned off because there are many forms of error reporting that involve too many modifications
